birds/js/birds.js

```diff
@@ -0,0 +1,166 @@
+            <p><strong>Found in:</strong> ${bird.found}</p>
+        `;
+        container.appendChild(animalInfo);
+    }
+}
+
+function myFunction(button) {
+    const animalInfo = button.closest('.animal-info');
+    const dots = animalInfo.querySelector('#dots');
+    const moreText = animalInfo.querySelector('#more');
+
+    if (dots.style.display === "none") {
+        dots.style.display = "inline";
+        button.innerHTML = "Read more";
+        moreText.style.display = "none";
+    } else {
+        dots.style.display = "none";
+        button.innerHTML = "Read less";
+        moreText.style.display = "inline";
+    }
+}
+
+const onClick = {
+    birdsClicked: false,
+  };
+  
+  const showBirdsWelcomeMessage = () => {
+    const welcomeMessageContainer = document.getElementById("home-message");
+  
+    if (!onClick.birdsClicked) {
+      welcomeMessageContainer.innerHTML = `
+        <h2>Hi and Welcome to the Birds page!</h2>
+        <p>We have three different birds here at the Zoo. To find more information about them, click on the button on the left side in the sidebar to find out more!</p>
+      `;
+      onClick.birdsClicked = true;
+    } else {
+      welcomeMessageContainer.innerHTML = "";
+    }
+  };
+
+  const birdsButton = document.getElementById("birds-button");
+  birdsButton.addEventListener("click", function (event) {
+    event.preventDefault();
+    showBirdsWelcomeMessage();
+  });
+  
+  document.querySelectorAll('a:not(#birds-button)').forEach(link => {
+    link.addEventListener('click', () => {
+      const welcomeMessageContainer = document.getElementById("home-message");
+      welcomeMessageContainer.innerHTML = "";
+      onClick.birdsClicked = false;
+    });
+  });
+
+const sidebarButtons = document.querySelectorAll('.animal-button');
+sidebarButtons.forEach(button => {
+  button.addEventListener('click', (event) => {
+    const birdName = event.target.getAttribute('data-animal');
+    displaySpecificBirdInfo(birdName);
+    onClick.showWelcomeMsg = false;
+  });
+});
+
+const sidebar = document.getElementById('sidebar');
+const toggleButton = document.getElementById('toggle-sidebar-btn');
+
+function toggleSidebar() {
+    sidebar.classList.toggle('close');
+    toggleButton.classList.toggle('rotate');
+    closeAllSubMenus();
+}
+
+function toggleSubMenu(button) {
+    if (!button.nextElementSibling.classList.contains('show')) {
+        closeAllSubMenus();
+    }
+
+    button.nextElementSibling.classList.toggle('show');
+    button.classList.toggle('rotate');
+
+    if (sidebar.classList.contains('close')) {
+        sidebar.classList.toggle('close');
+        toggleButton.classList.toggle('rotate');
+    }
+}
+
+function closeAllSubMenus() {
+    Array.from(sidebar.getElementsByClassName('show')).forEach(ul => {
+        ul.classList.remove('show');
+        ul.previousElementSibling.classList.remove('rotate');
+    });
+}
+
+toggleButton.addEventListener('click', toggleSidebar);
```

Comment:
Good attempt , It would be great if you write some comment with with function. just one line comment. Otherwise good work .